### PR TITLE
fix: Update Documentation badge link

### LINKED ISSUE #1067: Documentation badges link broken
**Describe the bug**
In the project's documentation, there are badges links are broken, leading to missing visual content.

https://github.com/konveyor/move2kube/blob/8aa892e2b4b50eaa67073358dbc85f6988dd7f3c/README.md?plain=1#L70

**To Reproduce**
1. Go to https://github.com/konveyor/move2kube/blob/main/README.md
2. Scroll down to Development environment setup section

**Expected behavior**
visual content should visible.

**Desktop (please complete the following information):**
 - OS: [e.g. RHEL]
 - Version [e.g. 22]. `move2kube version` should return it.

### PR DESCRIPTION
**Pull request by Sumitmaithani**
fixed #1067

### CODE REVIEW

**Comment by seshapad** [comment]
@Sumitmaithani Please change the PR title to include one of these ['feat', 'fix', 'docs', 'style', 'refactor', 'perf', 'test', 'build', 'ci', 'chore', 'revert']. The automated build check for the PR will validate it and issue error if not specified.

**Comment by seshapad** [comment]
> @Sumitmaithani Please change the PR title to include one of these ['feat', 'fix', 'docs', 'style', 'refactor', 'perf', 'test', 'build', 'ci', 'chore', 'revert']. The automated build check for the PR will validate it and issue error if not specified.

there is space after "fix" and before ":". please remove that.

**Comment by Sumitmaithani** [comment]
> > @Sumitmaithani Please change the PR title to include one of these ['feat', 'fix', 'docs', 'style', 'refactor', 'perf', 'test', 'build', 'ci', 'chore', 'revert']. The automated build check for the PR will validate it and issue error if not specified.
> 
> there is space after "fix" and before ":". please remove that.

Oh ! my mistake, thanks

**Comment by seshapad** [comment]
> > > @Sumitmaithani Please change the PR title to include one of these ['feat', 'fix', 'docs', 'style', 'refactor', 'perf', 'test', 'build', 'ci', 'chore', 'revert']. The automated build check for the PR will validate it and issue error if not specified.
> > 
> > 
> > there is space after "fix" and before ":". please remove that.
> 
> Oh ! my mistake, thanks

no worries. thank you.

File: README.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Checkout the [Tutorials](https://move2kube.konveyor.io/tutorials) and [Documenta
 
 ## Development environment setup
 
-To browse code [![Open in VSCode](https://open.vscode.dev/badges/open-in-vscode.svg)](https://open.vscode.dev/konveyor/move2kube)
+To browse code  [![Open in VSCode](https://badgen.net/badge/icon/visualstudio?icon=visualstudio&label)](https://open.vscode.dev/konveyor/move2kube)
```
Comment (Akash-Nayak) [inline README.md:70]: Can you please change `https://badgen.net/badge/icon/visualstudio?icon=visualstudio&label` to `https://badgen.net/badge/icon/Visual%20Studio%20Code?icon=visualstudio&label`

**Review by Akash-Nayak — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Sumitmaithani) [reply README.md]: updated 👍

**Review by Akash-Nayak — APPROVED** [review]
LGTM

**Review by HarikrishnanBalagopal — APPROVED** [review]
LGTM

**Pull request merged**